torch/distributed/distributed_c10d.py

```diff
@@ -463,9 +462,6 @@ def init_process_group(backend,
             backend, ``is_high_priority_stream`` can be specified so that
             process group can pick up high priority cuda streams.
 
-    .. note:: Note that if passing in pg_options and set the ``pg_options.timeout``,
```

Comment:
Above for the doc about `pg_options`, should we mention that only `ProcessGroupNCCLOptions` should be used?